packages/libdash/libdash.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "libdash"
+version: "0.1"
+synopsis: "Bindings to the dash shell's parser"
+maintainer: "Michael Greenberg <michael.greenberg@pomona.edu>"
+authors: "Michael Greenberg <michael.greenberg@pomona.edu>"
+license: "BSD"
+homepage: "https://github.com/mgree/libdash"
+bug-reports: "https://github.com/mgree/libdash/issues"
+depends: [
+  "ocaml" {>= "4.0.7"}
+  "ctypes" {= "0.11.5"}
+  "ctypes-foreign" {>= "0.4.0"}
+  "conf-autoconf" {build}
+]
+build: [
+  ["./autogen.sh"]
+  ["./configure" "--prefix=%{prefix}%" "--libdir=%{lib}%"]
+  [make]
+]
+install: [
+  [make "install"]
+  [make "-C" "ocaml"]
```

Comment:
Is this command building something? If so this should be in the `build` section, not in the `install` section

Reply:
Looking at the makefile I think it is the case. I'll fix that